python/tests/async_tests/test_fork_safety.py

```
@@ -0,0 +1,186 @@
+            # Issue several commands to exercise the pipe transport
+            pid = os.getpid()
+            for i in range(10):
+                await client.set(f"fork_test:{pid}:{i}", f"value-{i}")
+
+            # Verify reads work too
+            result = await client.get(f"fork_test:{pid}:0")
+            assert result == b"value-0", f"Expected b'value-0', got {result!r}"
+
+            await client.close()
+            result_queue.put(("OK", pid))
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)